apps/onboarding/src/session/session.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common'
+import { v4 as uuidv4 } from 'uuid'
+import { Session } from './session.entity'
+import { SessionRepository } from './session.repository'
+
+
+@Injectable()
+export class SessionService {
+
+  constructor(private readonly sessionRepository: SessionRepository,) {}
+
+  async createSession(externalAccount: string): Promise<Session> {
+    const session = Session.of({
+      id: uuidv4(),
+      externalAccount: externalAccount,
+      createdAt: new Date(Date.now()).toISOString(),
+      expiredAt: new Date(Date.now()+ 3600).toISOString(),
```

Comment:
`3600` is 3.6 seconds which seems way too short for a session. Given that we're recording the `createdAt` time, we can calculate the expiry when we read the record, which gives us more flexibility for forward compatibility.

Reply:
Good catch, I missed that in the review, indeed we shouldn't set expiry on creation.